server/swagger.json

```diff
@@ -3999,6 +3999,13 @@
 									"$ref": "#/definitions/DashboardQuery"
 								}
 							},
+              "axes": {
+                "description": "The viewport for a cell's visualizations",
+                "type": "object",
+                "additionalProperties": {
```

Comment:
I think that we'll have those as well for the near-term, however this API is flexible enough to represent n-dimensional visualizations should we ever do anything with WebGL in the future.